hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestGcsEventsHoodieIncrSource.java

```diff
@@ -135,10 +128,10 @@ public void shouldNotFindNewDataIfCommitTimeOfWriteAndReadAreEqual() throws IOEx
 
     Pair<String, List<HoodieRecord>> inserts = writeGcsMetadataRecords(commitTimeForWrites);
 
-    readAndAssert(READ_UPTO_LATEST_COMMIT, Option.of(commitTimeForReads), 100L, 0, inserts.getKey());
+    readAndAssert(READ_UPTO_LATEST_COMMIT, Option.of(commitTimeForReads), 100L, inserts.getKey());
 
     verify(gcsObjectMetadataFetcher, times(0)).getGcsObjectMetadata(Mockito.any(), Mockito.any(),
```

Comment:
Since we are moving away from mocked `GcsObjectMetadataFetcher`. Can we remove the field `gcsObjectMetadataFetcher`?

Reply:
@lokesh-lingarajan-0310 I think this was missed in the update.